src/convertToObject.js

```diff
@@ -7,6 +7,16 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString.split(';').reduce((styles, styleString) => {
+    const updatedStyles = { ...styles };
+    const styleKeyValue = styleString.split(':');
+
+    if (styleKeyValue[0] && styleKeyValue[1]) {
+      updatedStyles[styleKeyValue[0].trim()] = styleKeyValue[1].trim();
+    }
+
+    return updatedStyles;
+  }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable. For better readability, consider breaking the chain into multiple lines.